libs/newsletter/guides/installation.md

```diff
@@ -0,0 +1,13 @@
+# Installation
+
```

Comment:
Add a section for per-platform installation requirements (e.g. in-memory, etc).

Also add a Table of Contents like:

Table of Contents
====

NPM
Yarn
Platorms:
* in-memory-web-api
* Magento (coming soon)
* Shopify (coming soon)